.github/workflows/main.yml

```diff
@@ -0,0 +1,96 @@
+name: Main
+
+on:
+  push:
+    branches: [ main, wip ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Every twelve (12) hours.
+    # @see https://crontab.guru/every-12-hours
+    - cron: 0 */12 * * *
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: "${{ matrix.os }}-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu ]
+        php: [ 8.1, 8.2, 8.3 ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4 # https://github.com/marketplace/actions/checkout
+
+      - name: Set up PHP CLI
+        uses: shivammathur/setup-php@v2 # https://github.com/marketplace/actions/setup-php-action
+        with:
+          php-version: "${{ matrix.php }}"
+
+      - name: Create project
+        run: |
+          set -o xtrace
+
+          # Prepare the target directory.
+          sudo mkdir -p /var/www
+          sudo rm -rf /var/www/*
+          sudo chmod 777 /var/www
+
+          composer create-project \
+            --stability=dev \
+            --repository='{"type":"vcs","url":"'git@github.com:phenaproxima/drupal-suite.git'"}' \
+            --ansi \
+            drupal/starshot-project \
+            /var/www
+
+          # Enable verbose error logging (debug mode).
+          echo "if (file_exists(__DIR__ . '/settings.local.php')) {
+            include __DIR__ . '/settings.local.php';
+          }" | sudo tee -a /var/www/web/sites/default/settings.php
+          sudo cp /var/www/web/sites/example.settings.local.php \
+            /var/www/web/sites/default/settings.local.php
```

Comment:
We don't need this. It should be done by a Drush command run after installation.

Reply:
Should be. My point is that you can set the error logging level as a config setting in the database; doing it by `settings.local.php` is tricky overkill. All you need to do is:

```
./vendor/bin/drush config:set --yes system.logging error_level verbose
```

Reply:
That works to set the config, but for some reason, it doesn't take effect later when I `curl` the site. So I'm just putting a `@todo` in there for the time being, until we figure out why not.